test/manual/amp-selector.amp.html

```diff
@@ -0,0 +1,120 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>AMP #0</title>
+  <link rel="canonical" href="amps.html" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <link href='https://fonts.googleapis.com/css?family=Questrial' rel='stylesheet' type='text/css'>
+  <script async custom-element="amp-form" src="../../dist/v0/amp-form-0.1.max.js"></script>
+  <script async custom-element="amp-selector" src="../../dist/v0/amp-selector-0.1.max.js"></script>
+  <script async custom-element="amp-carousel" src="../../dist/v0/amp-carousel-0.1.max.js"></script>
+  <style amp-custom>
+  /* AMP.CSS */
+  html, body {
+    overflow-x: hidden !important;
+    height: auto !important;
+  }
+
+  /**
+   * Margin:0 is currently needed for iOS viewer embeds.
+   * See:
+   * https://github.com/ampproject/amphtml/blob/master/spec/amp-html-layout.md
+   * and {@link ViewportBindingNaturalIosEmbed_} for more info.
+   */
+  body {
+    margin: 0 !important;
+  }
+  /* END AMP.CSS */
+
+  amp-selector *[option] {
+    box-sizing: border-box !important;
+    display: inline-block;
+    transition: border 0.30s ease-in-out;
+    border: 1px solid transparent;
+    position: relative !important;
+  }
+
+  amp-selector *[selected] {
```

Comment:
No need for `*`. But actually this would be better: `.myitem[selected] {}`.

Also, we may want to consider the default very basic style. E.g. using `outline`. It's very easy to cancel, but the element is immediately visually usable.